corehq/apps/export/views/list.py

```diff
@@ -1047,6 +1089,134 @@ def page_context(self):
+        if user_can_view_deid_exports(self.domain, self.request.couch_user):
+            return 'deid'
+        return 'neither'
+
+    @memoized
+    def get_saved_exports(self):
+        if self.allowed_doc_type == 'neither':
+            return []
+
+        if self.allowed_doc_type == 'deid':
+            exports = get_brief_deid_exports(self.domain, None)
+        else:
+            exports = get_brief_exports(self.domain, self.allowed_doc_type)
+        return [x for x in exports if self._should_appear_in_list(x)]
+
+    @property
+    def create_export_form(self):
+        form = CreateExportTagForm(True, True)
+        form.fields['model_type'].label = _("Sheet Type")
+
+        model_type_choices = [
+            ('', _("Select field type")),
+        ]
+        if self.has_case_export_permissions:
+            model_type_choices.append(('case', _('Case')))
+        if self.has_form_export_permissions:
+            model_type_choices.append(('form', _('Form')))
+        form.fields['model_type'].choices = model_type_choices
+
+        return form
+
+    @property
+    @memoized
+    def live_google_sheets_limit(self):
+        try:
+            domain_object = ProjectLimit.objects.get(domain=self.domain)
+            return domain_object.limit_value
+        except ProjectLimit.DoesNotExist:
+            return settings.DEFAULT_GSHEET_LIMIT
+
+    @property
+    def create_export_form_title(self):
+        return _("Select Model Type")
+
+    def _should_appear_in_list(self, export):
+        test = IntegrationFormat.is_integration_format(export['export_format'])
+        return test
+
+    def fmt_export_data(self, export):
+        data = super(LiveGoogleSheetListHelper, self).fmt_export_data(export)
+        data.update({
+            'isGoogleSheet': True,
+        })
+        if len(self.get_saved_exports()) >= self.live_google_sheets_limit:
+            data['editUrl'] = '#IntegratedExportLimitReachedModal'
+        return data
+
+    def _download_view(self, export):
+        # If this isn't added in it will cause the listview to not populate. Will not show up in the UI
+        from corehq.apps.export.views.download import DownloadNewCaseExportView, DownloadNewFormExportView
+        if isinstance(export, FormExportInstance):
+            return DownloadNewFormExportView
+        return DownloadNewCaseExportView
+
+
+@location_safe
+class LiveGoogleSheetListView(BaseExportListView, LiveGoogleSheetListHelper):
+    is_live_google_sheet = True
+    urlname = 'list_live_google_sheets'
+    page_title = "Live Google Sheet Integration"
+
+    @flag_enabled('GOOGLE_SHEETS_INTEGRATION')
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
+
+    # TODO insert link to confluence page
+    @property
+    def lead_text(self):
+        return format_html(_("""
+        Integrate your live CommCare data with Google Sheets.
+        <a href=""
+       id="js-gsheet-track-learn-more"
```

Comment:
nit: usually attributes are indented so that they are all in line with the first attribute used (`href`)